extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php

```diff
@@ -36,9 +37,10 @@ public function __construct() {
 	public function postMessage( int $siteId, string $language ): bool {
 		$transformedLang = $this->getTransformedLang( $language );
 		$staffId = $this->getStaffFromLang( $transformedLang );
-		$message = $this->getMessage( $transformedLang );
+		$message = $this->getBodyMessage( $transformedLang );
+		$title = $this->getTitleMessage( $transformedLang );
 
-		$success = $this->threadCreator->create( $staffId, $siteId, $message );
+		$success = $this->threadCreator->create( $staffId, $siteId, $message, $title );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$success". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)